tests/pulsar/test_client.py

```diff
@@ -0,0 +1,70 @@
+# Tests adapted from Apache Pulsar tests:
+# https://github.com/apache/pulsar-client-python/blob/main/tests/pulsar_test.py
+import subprocess
+from unittest import main
+
+import pytest
+from _pulsar import Timeout
+
+from eodhp_utils.pulsar.client import Consumer, Producer
+
+timeout_ms = 2000  #
+service_url = "pulsar://localhost:6650"
+
+
+@pytest.fixture(autouse=True)
+def environment_variables(monkeypatch):
+    monkeypatch.setenv("PULSAR_URL", service_url)
```

Comment:
don't rely on env vars in unit tests, they can't handle concurrency. They are effectively globals, and globals are bad outside of some very niche use cases.